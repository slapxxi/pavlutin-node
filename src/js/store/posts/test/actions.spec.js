import {
  addPost,
  requestPosts,
  requestPostsError,
} from '../actions';

test('addPost returns action', () => {
  expect(addPost()).toMatchSnapshot();
});

test('requestPosts returns action', () => {
  expect(requestPosts()).toMatchSnapshot();
});

test('requestPostsError returns action', () => {
  expect(requestPostsError({})).toMatchSnapshot();
});
