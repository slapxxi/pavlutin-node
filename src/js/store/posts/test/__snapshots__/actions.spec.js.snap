// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`addPost returns action 1`] = `
Object {
  "payload": undefined,
  "type": "ADD_POST",
}
`;

exports[`requestPosts returns action 1`] = `
Object {
  "type": "REQUEST_POSTS",
}
`;

exports[`requestPostsError returns action 1`] = `
Object {
  "payload": Object {},
  "type": "REQUEST_POSTS_ERROR",
}
`;
